crates/red_knot_python_semantic/resources/mdtest/type_properties/is_equivalent_to.md

```diff
@@ -32,4 +32,14 @@ static_assert(not is_equivalent_to(Literal[1, 0], Literal[1, 2]))
 static_assert(not is_equivalent_to(Literal[1, 2, 3], Literal[1, 2]))
 ```
 
+## `object & ~T` is equivalent to `~T`
+
+```py
+from knot_extensions import Intersection, Not, is_equivalent_to, static_assert
+
+class P: ...
+
+static_assert(is_equivalent_to(Intersection[object, Not[P]], Not[P]))
+```
```

Comment:
Maybe this could go in `intersection_types.md` instead? Maybe below the "`Never` in intersections" section, as a new "`object` in intersections" section?

Reply:
Ah yes, that's a better place for this, thanks!